quic/client/handshake/FizzClientHandshake.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+#pragma once
+
+#include <quic/client/handshake/ClientHandshake.h>
+
+namespace quic {
+
+class FizzClientContext;
+
+class FizzClientHandshake : public ClientHandshake {
+ public:
+  FizzClientHandshake(
+      QuicCryptoState& cryptoState,
+      std::shared_ptr<FizzClientContext> fizzContext);
+
+ private:
+  std::shared_ptr<FizzClientContext> fizzContext_;
```

Comment:
Feedback from @yangchi, where is thi actually used? In the next diff it also seems unused and the `FizzClientContext` is simply passed in to connect.

Reply:
I can push more of the stack if you want. After connect is moved down, the next step is to change its API to request FizzClientContext for fizz specific stuff.